common/services/base.js

```diff
@@ -1,10 +1,18 @@
+const { isObject, isString, isNil, isEmpty, omitBy } = require('lodash')
+
 const ApiClient = require('../lib/api-client')
 
 class BaseService {
   constructor(req) {
     this.req = req || {}
     this.apiClient = (req && req.apiClient) || new ApiClient()
   }
+
+  removeInvalid(input) {
```

Comment:
Is this really part of the `BaseService` or is it more of a general utility method? It looks like it could be used elsewhere quite easily.

Reply:
I toyed with where this should live....the reason I ended up putting it here is because it made it easier to finish the tests for this set of changes and also it's more specifically linked to these services.

Reply:
It feels more like a https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/tree/master/common/helpers to me than a core `BaseService` method - but that does make it a little more annoying to use

Reply:
Yes, I think that was the decision I took at the time. This work has taken quite a while and I didn't want to waste longer refactoring things again too much. 

The other reason for placing it here is that it's only related to these specific services and we didn't currently have any service related "helpers" yet either so it wasn't going to create two places for the same thing.